root/edit/EditData.js

```diff
@@ -0,0 +1,105 @@
+/*
+ * @flow strict-local
+ * Copyright (C) 2021 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import Layout from '../layout';
+import EditLink from '../static/scripts/common/components/EditLink';
+import EntityLink from '../static/scripts/common/components/EntityLink';
+import commaOnlyList from '../static/scripts/common/i18n/commaOnlyList';
+import {bracketedText} from '../static/scripts/common/utility/bracketed';
+import {formatPluralEntityTypeName}
+  from '../static/scripts/common/utility/formatEntityTypeName';
+import {getEditStatusName} from '../utility/edit';
+
+type Props = {
+  +edit: {...EditT, +id: number},
+  +relatedEntities: {[type: CoreEntityTypeT]: {
+    [entityId: string]: CoreEntityT,
+  }},
+};
+
+const EditIndex = ({
+  edit,
+  relatedEntities,
+}: Props): React.Element<typeof Layout> => {
+  const title = texp.l('Edit data for edit #{id}', {id: edit.id});
+  const jsonData = JSON.parse(edit.raw_data);
+  const displayData = jsonData
+    ? JSON.stringify(jsonData, null, 2)
+    : edit.raw_data;
```

Comment:
Not sure I understand this part.  How come we need to send `raw_data` to do `JSON.parse(edit.raw_data)` instead of just using `edit.data`?

In the next line, how come a ternary is needed to check `jsonData`?

Reply:
I was just copying what `root/edit/data.tt` used, since I assumed it was useful, but I guess if edit.data is already JSON, that might be easier :)